tests/phpunit/tests/Story_Renderer.php

```diff
@@ -24,6 +24,9 @@ class Story_Renderer extends \WP_UnitTestCase {
 
 	public static function wpSetUpBeforeClass( $factory ) {
 		self::$user = $factory->user->create( [ 'role' => 'administrator' ] );
+		if ( is_multisite() ) {
```

Comment:
@swissspidy This is something we will have to look at some point. Unfiltered html is only a super admin cap in multisite.

Reply:
Agreed. This shouldn't be needed now that we have all these KSES fixes